src/many/src/server/module/_7_kvstore_commands/delete.rs

```diff
@@ -8,6 +9,4 @@ pub struct DeleteArgs {
     pub key: ByteVec,
 }
 
-#[derive(Encode, Decode)]
-#[cbor(map)]
-pub struct DeleteReturns {}
+pub type DeleteReturn = EmptyReturn;
```

Comment:
Not sure I understand this one. Care to shed some light?

The `delete` return value in the spec is `{}`. If I understand correctly, this change would imply that the return value of `delete` is an empty map?

Reply:
Delete is NOT `{}`, it's `()` which is anything.

Reply:
We encode it as a value so it doesn't break decoders that would expect one, but it shouldn't have any value, really. Empty returns are 0-byte semantically, but we don't want to break CBOR.